### ISSUE TITLE
Documentation is not built/installed

### ISSUE THREAD

**Issue by mikrosk** [freemint/cops#1]
Although travis does install both HCP & UDO, `doc/Makefile` is not executed (see https://github.com/freemint/toswin2 where it works).

**Comment by th-otto** [comment]
Yes, i know. See also the comment of commit https://github.com/freemint/cops/commit/c2bd9ab3f36f4726aa1cc8a598e5ae00004c8cd8

We also have to decide where to install them for the snapshots. Should that be `<sysroot>`/guides? And what about different languages?

But please do me a favour. Move the build scripts from the travis-scripts repo to this one. Having them in different repos is totally crap.

**Comment by mikrosk** [comment]
Already done, see for yourself! :)

**Comment by mikrosk** [comment]
> We also have to decide where to install them for the snapshots. Should that be <sysroot>/guides? And what about different languages?

I wouldn't worry about it, just put it in `doc` folder. This kind of issues should be solved at the "distribution" level, i.e. in freemint snapshots. There we can take the usual route, copy english HYP into `guides` and let the rest in the application folder, similar what we do with other tools.

**Comment by th-otto** [comment]
Yes,  but "make install" should copy it somewhere. Preferably to some place where utils.sh for the snapshots can pick it up ;)

**Comment by mikrosk** [comment]
As I say, a `$installdir/doc` and `$installdir/<language>` is totally fine.
